README.rst

```diff
@@ -5,31 +5,179 @@ Grafana
 
 A beautiful, easy to use and feature rich Graphite dashboard replacement and graph editor.
 
+
 Sample pillars
 ==============
 
-Sample pillar installed from system package
+
+Server deployments
+------------------
+
+Server installed from system package
+
+.. code-block:: yaml
+
+    grafana:
+      server:
+        enabled: true
+        admin:
+          user: admin
+          password: passwd
+        database:
+          engine: sqlite
+
+Server installed with PostgreSQL database
+
+.. code-block:: yaml
 
     grafana:
       server:
         enabled: true
+        admin:
+          user: admin
+          password: passwd
         database:
           engine: postgresql
           host: localhost
           port: 5432
-        data_source:
-          metrics1:
-            engine: graphite
-            host: metrics1.domain.com
-            ssl: true
+          name: grafana
+          user: grafana
+          password: passwd
+
+
+Collector setup
+---------------
+
+Used to aggregate dashboards from monitoring node.
+
+.. code-block:: yaml
+
+    grafana:
+      collector:
+        enabled: true
+
+
+Client setups
+-------------
+
+Client enforced data sources
+
+.. code-block:: yaml
+
+    grafana:
+      client:
+        enabled: true
+        server:
+          protocol: https
+          host: grafana.host
+          port: 3000
+          token: token
+        datasource:
+          graphite:
+            type: graphite
+            host: mtr01.domain.com
+            protocol: https
             port: 443
-            user: test
-          metrics2:
-            engine: elasticsearch
-            host: metrics2.domain.com
+          elasticsearch:
+            type: elasticsearch
+            host: log01.domain.com
             port: 80
-            user: test
+            user: admin
+            password: password
```

Comment:
nit but unless you use Shield or some kind of reverse proxy, Elasticsearch doesn't require authentication.